nav2_bringup/launch/localization_launch.py

```diff
@@ -102,6 +101,7 @@ def generate_launch_description():
     load_nodes = GroupAction(
         condition=IfCondition(PythonExpression(['not ', use_composition])),
         actions=[
+            SetParameter("use_sim_time", use_sim_time),
```

Comment:
does SetParameter work globally or can it be used for a single node? Also, does that work if the parameter in question is inside of the yaml file also (clearly it works if its not inside the yaml file)? I ask because this could be a really nice solution for the `yaml_filename` thing too if we can set it for only the map server if there is a non-trivial value set in the launch configuration.

Reply:
> does SetParameter work globally or can it be used for a single node?

It applies the parameter for all nodes within the GroupAction. See [here](https://github.com/ros2/launch_ros/blob/5434428a30a981aeee1dbadfd5b43622cef12d67/launch_ros/launch_ros/actions/set_parameter.py#L37)

> Also, does that work if the parameter in question is inside of the yaml file also

yes, the parameter in the yaml would override

For the refactoring of `yaml_filename`, I'd prefer if we can open a separate issue. I might take it but can't commit right now

Reply:
Totally understood, just asking the question, it sounds like this would be a solution for it too. Duly noted.

Reply:
Leaving this un-resolved as a reminder to myself